gcp/kubernetes/run_batch.sh

```diff
@@ -8,7 +8,7 @@
 #    - the resources defined in templates/jiant_env.libsonnet are correct
```

Comment:
Is this just a copy of your other PR? If so, merge that first and confirm no conflicts.

Reply:
Done.